lexicons/app/bsky/feed/defs.json

```diff
@@ -41,7 +41,8 @@
         "repost": { "type": "string", "format": "at-uri" },
         "like": { "type": "string", "format": "at-uri" },
         "threadMuted": { "type": "boolean" },
-        "replyDisabled": { "type": "boolean" }
+        "replyDisabled": { "type": "boolean" },
+        "quotepostDisabled": { "type": "boolean" }
```

Comment:
In lexicon quote posts are expressed as "post record embeds."  So I'm wondering if `embedDisabled` would be more consistent here.